entrypoint.sh

```diff
@@ -1,30 +1,98 @@
 #! /usr/bin/env bash
 
-set -eo pipefail
+set -euo pipefail
```

Comment:
Let's do even better:
```suggestion
set -Eeuo pipefail
```